packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -380,15 +380,16 @@ class _ModalBottomSheetRoute<T> extends PopupRoute<T> {
 
   @override
   Widget buildPage(BuildContext context, Animation<double> animation, Animation<double> secondaryAnimation) {
+    final BottomSheetThemeData sheetTheme = theme?.bottomSheetTheme ?? Theme.of(context).bottomSheetTheme;
```

Comment:
Could this be handled with InheritedTheme.captureAll() instead?

Reply:
Not sure exactly what you are thinking of here, but `theme` is a passed in parameter that wouldn't necessarily be in a `Theme` widget, so its values wouldn't be seen by `captureAll()`.  Or am I misunderstanding what you suggesting?

Reply:
What I was suggesting would have required adding a Builder around the new page, so that you could bind a variable to Theme.of(context).bottomSheetTheme, which doesn't seem to be worth trouble. Never mind :-).